lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/Transactions.hs

```diff
@@ -229,7 +263,79 @@ spec = do
             , expectResponseCode HTTP.status400
             ]
 
+    it "TRANS_EXTERNAL_CREATE_03 - empty payload" $ \ctx -> do
+        _ <- emptyWallet ctx
+        let headers = Headers [ ("Content-Type", "application/octet-stream") ]
+        r <- request @ApiTxId ctx postExternalTxEp headers Empty
+        verify r
+            [ expectErrorMessage errMsg400MalformedTxPayload
+            , expectResponseCode HTTP.status400
+            ]
+
+    describe "TRANS_EXTERNAL_CREATE_04 - \
+        \v2/external-transactions - Methods Not Allowed" $ do
+
+        let matrix = ["PUT", "DELETE", "CONNECT", "TRACE", "OPTIONS", "GET"]
+        forM_ matrix $ \method -> it (show method) $ \ctx -> do
+            w <- emptyWallet ctx
+            addr:_ <- listAddresses ctx w
+            let addrStr = encodeAddress (Proxy @t) (getApiT $ fst $ addr ^. #id)
+
+            txBlob <- prepExternalTxViaJcli addrStr 1
+            let payload = (NonJson . BL.fromStrict . toRawBytes Base16) txBlob
+            let headers = Headers [ ("Content-Type", "application/octet-stream") ]
+
+            let endpoint = "v2/external-transactions"
+            r <- request @ApiTxId ctx (method, endpoint) headers payload
+            expectResponseCode @IO HTTP.status405 r
+            expectErrorMessage errMsg405 r
+
+    describe "TRANS_EXTERNAL_CREATE_04 - HTTP headers" $ do
+        forM_ (externalTxHeaders @ApiTxId)
+            $ \(title, headers, expectations)
+            -> it title $ \ctx -> do
+
+            w <- emptyWallet ctx
+            addr:_ <- listAddresses ctx w
+            let addrStr = encodeAddress (Proxy @t) (getApiT $ fst $ addr ^. #id)
+
+            txBlob <- prepExternalTxViaJcli addrStr 1
+            let payload = (NonJson . BL.fromStrict . toRawBytes Base16) txBlob
+
+            r <- request @ApiTxId ctx postExternalTxEp headers payload
+            verify r expectations
+
   where
+
+    externalTxHeaders
+        :: (Show a)
+        => [( String
+            , Headers
+            , [(HTTP.Status, Either RequestException a) -> IO ()])
+           ]
+    externalTxHeaders =
+        [
+        -- ( "No HTTP headers -> 415"
+        --   , None
+        --   , [ expectResponseCode @IO HTTP.status202 ]
```

Comment:
So not providing any headers (no `Content-Type: application\octet-stream` in particular) but providing proper binary to the endpoint it seems to be processed with `202`... I was expecting `415` since no content-type provided. Not sure if that's a problem...

Reply:
maybe it takes `octet-stream` as default when not specified. But not sure here

Reply:
It doesn't feel like it. (https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core/src/Cardano/Wallet/Api.hs#L154-L156). :man_shrugging: